coderwall.svg

```diff
@@ -1,58 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-
-<svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   version="1.1"
-   width="28"
-   height="27"
-   id="svg3039">
-  <metadata
-     id="metadata3045">
-    <rdf:RDF>
-      <cc:Work
-         rdf:about="">
-        <dc:format>image/svg+xml</dc:format>
-        <dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
-        <dc:title></dc:title>
-      </cc:Work>
-    </rdf:RDF>
-  </metadata>
-  <defs
-     id="defs3043" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,-1.3021943,-1.5935064)"
-     id="path3049"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,8.2978057,-1.5935064)"
-     id="path3049-3"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,17.897806,-1.5935064)"
-     id="path3049-35"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,8.2978057,7.0064936)"
-     id="path3049-3-0"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,17.897806,7.0064936)"
-     id="path3049-35-7"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889236,0,0,0.70313852,17.897806,15.606494)"
-     id="path3049-35-6"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Drawn from scratch, based on coderwall.png from https://assembly.com/coderwall/assets -->
+<svg version="1.1" viewBox="0 0 300 300" xmlns="http://www.w3.org/2000/svg">
+<circle cx="30" cy="30" r="30" fill="#3e8dcc"/>
```

Comment:
Please indent these `circle` tags. I prefer XML tags to have indentation for each level of depth.